swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -14,9 +16,13 @@ enum PacketTunnelProviderError: String, Error {
 }
 
 class PacketTunnelProvider: NEPacketTunnelProvider {
-  static let logger = Logger(subsystem: "dev.firezone.firezone", category: "packet-tunnel")
+  let logger = Logger(subsystem: "dev.firezone.firezone", category: "packet-tunnel")
 
-  private lazy var adapter = Adapter(with: self)
+  // Till connlib is updated to expose the desired FFI, we'll use a mock
+  private lazy var adapter = ConnlibMock.Adapter(with: self)
```

Comment:
Couldn't this be made configurable or something? It's important to be able to test connlib with the Apple client; if a specific pending FFI change is blocking this I can address that ASAP.

Reply:
That's a good question -- probably better suited for @roop. This was just copy-pasta'd from the `demoable_macos_app` branch in the firezone-apple repo.

I don't think this PR needs merging urgently; I'll mark it as draft and defer to @roop. This was mainly to get the updated UI into TestFlight, but that process is decoupled from our CI pipeline at the moment.

Reply:
I've reverted everything except the UI changes and it still builds fine, so I'll leave it like and see if @roop wants to make changes.

Reply:
I've added commits to make the status bar menu (macOS) update when connlib updates its resources .